recipes/cuda-cudart/meta.yaml

```
@@ -0,0 +1,140 @@
+  sha256: d785fd613e35e3761fdb0016a31a56417f570605cbfb9a93ca9dc68be6789188  # [ppc64le]
+  sha256: 5de248c6c203f6c91ce34256bbfc028bc87c1b0c552f9f51edb65e60ad665f63  # [aarch64]
+  sha256: 1cd8516feee068c6d718453e76b0dfcbee27844a35d4f3aa608a3d316190309c  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:                # [linux]
+    strong:                   # [linux]
+      - __glibc >=2.17        # [linux]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+  run:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+
+outputs:
+  - name: cuda-cudart
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcudart.so.{{ version }}     # [linux]
+        - if not exist %LIBRARY_BIN%\cudart64_*.dll exit 1   # [win]
+    files:
+      - lib/*.so.*                # [linux]
+      - targets/*/lib/*.so.*      # [linux]
+      - Library\bin               # [win]
+
+  - name: cuda-cudart-dev
+    run_exports:
+      weak:
+        - {{ pin_subpackage("cuda-cudart", max_pin='x') }}
+    requirements:
+      run:
+        - cuda-cudart >={{ version }}
+        - cuda-cudart-static >={{ version }}
+        - cuda-cudart-dev_{{ target_platform }} =={{ version }}  # [linux]
+    test:
+      commands:
+        - test -d $PREFIX/lib/pkgconfig                  # [linux]
+        - if not exist %LIBRARY_LIB%\cudart.lib exit 1   # [win]
+    files:
+      - lib/*.so                                # [linux]
+      - include                                 # [linux]
+      - lib/pkgconfig                           # [linux]
+      - Library\include                         # [win]
+      - Library\lib\cudadevrt.lib               # [win]
+      - Library\lib\cuda.lib                    # [win]
+      - Library\lib\cudart.lib                  # [win]
+
+  - name: cuda-cudart-dev_{{ target_platform }}
+    build:
+      skip: True                        # [not linux]
+    requirements:
+      run:
+        - cuda-cudart-dev_{{ target_platform }} >={{ version }}
+    test:
+      commands:
+        - test -L $PREFIX/targets/*/lib/libcudart.so
+    files:
+      - targets/*/lib/*.so
+      - targets/*/include/*
+      - targets/*/include/cooperative_groups/*
+
+  - name: cuda-cudart-static
+    requirements:
+      run:
+        - cuda-cudart-static_{{ target_platform }} =={{ version }}  # [linux]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcudart_static.a               # [linux]
+        - if not exist %LIBRARY_LIB%\cudart_static.lib exit 1  # [win]
+    files:
+      - lib/*.a                        # [linux]
+      - Library\lib\cudart_static.lib  # [win]
+
+  - name: cuda-cudart-static_{{ target_platform }}
+    build:
+      skip: True                        # [not linux]
```

Make this noarch so that it can be installed in other platforms. (For eg: need to install `cuda-cudart-static_linux-aarch64` when building from `linux-64`.)
```suggestion
    build:
      skip: True                        # [not linux]
      noarch: generic
```